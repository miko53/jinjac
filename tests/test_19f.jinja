{% if not_exist %}
This part is not displayed
{% for x in range(0,5) %}
--> loop inactive here {{ x }} and {{ x+1 }}
{% endfor %}
{% else %}
 active else {{ a_ident }}
{% for x in range(0,5) %}
--> loop here {{ x }} and {{ x+1 }}
{% endfor %}
{% endif %}
TOTO
