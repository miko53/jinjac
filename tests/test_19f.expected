
 active else TheValeur

--> loop here 0 and 1

--> loop here 1 and 2

--> loop here 2 and 3

--> loop here 3 and 4

--> loop here 4 and 5


TOTO
